Python/tigre/algorithms/iterative_recon_alg.py

```diff
@@ -352,7 +331,7 @@ def update_image(self, geo, angle, iteration):
         VERBOSE:
          for j in range(angleblocks):
              angle = np.array([alpha[j]], dtype=np.float32)
-             proj_err = proj[angle_index[j]] - Ax(res, geo, angle, 'Siddon')
+             proj_err = proj[angle_index[j]] - Ax(res, geo, angle, 'ray-voxel')

```

Comment:
```ray-voxel``` is supported but obsolete. Could you please revert this line?

Reply:
Done